### ISSUE TITLE
npyImport.swift:1:8: error: no such module 'SwiftNpy'

### ISSUE THREAD

**Issue by tgautam03** [qoncept/swift-npy#1]
Hi

I'm trying to use this packlage for one of my projects but I can't import it. I'm getting an error 
```
npyImport.swift:1:8: error: no such module 'SwiftNpy'
import SwiftNpy
```
I've added the repo's link in the dependencies section of `Package.swift` file and also ran `swift package resolve`.
```swift
// swift-tools-version:5.3
// The swift-tools-version declares the minimum version of Swift required to build this package.

import PackageDescription

let package = Package(
    name: "swiftDSP",
    products: [
        // Products define the executables and libraries a package produces, and make them visible to other packages.
        .library(
            name: "swiftDSP",
            targets: ["swiftDSP"]),
    ],
    dependencies: [
        // Dependencies declare other packages that this package depends on.
        .package(url: "https://github.com/qoncept/swift-npy.git", from: "0.1.0"),
    ],
    targets: [
        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
        // Targets can depend on other targets in this package, and on products in packages this package depends on.
        .target(
            name: "swiftDSP",
            dependencies: []),
        .testTarget(
            name: "swiftDSPTests",
            dependencies: ["swiftDSP"]),
    ]
)
```

I'm using Ubuntu 20.04 and new to swift. I hope you can help me out here.

Thanks

**Comment by t-ae** [comment]
Try this `Package.swift`.

```swift
// swift-tools-version:5.3
// The swift-tools-version declares the minimum version of Swift required to build this package.

import PackageDescription

let package = Package(
    name: "swiftDSP",
    products: [
        // Products define the executables and libraries a package produces, and make them visible to other packages.
        .library(
            name: "swiftDSP",
            targets: ["swiftDSP"]),
    ],
    dependencies: [
        // Dependencies declare other packages that this package depends on.
        // .package(url: /* package url */, from: "1.0.0"),
        .package(name: "SwiftNpy", url: "https://github.com/qoncept/swift-npy.git", from: "0.1.0")
    ],
    targets: [
        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
        // Targets can depend on other targets in this package, and on products in packages this package depends on.
        .target(
            name: "swiftDSP",
            dependencies: ["SwiftNpy"]),
        .testTarget(
            name: "swiftDSPTests",
            dependencies: ["swiftDSP"]),
    ]
)
```

There are two changes: 

1. Specifying `name: "SwiftNpy"` explicitly in dependencies section
2. Declaring `dependencies: ["SwiftNpy"]` in targets section

**Comment by tgautam03** [comment]
I modified the file but I'm getting the same error. I don't understand how I can use the package?

**Comment by tgautam03** [comment]
I also ran swift build and got several erorrs like `error: cannot find 'CFSwapInt16HostToBig' in scope
        uints = elements.map(CFSwapInt16HostToBig)`

**Comment by t-ae** [comment]
It turned out that 0.1.0 is not compilable on Linux environment.
I fixed the problem and released new version 0.1.1.
Please try this version.

**Comment by random-yang** [comment]
@tgautam03 So, is it finally worked?
